src/Training/Bundle/UserNamingBundle/Resources/config/services.yml

```
@@ -0,0 +1,23 @@
+services:
+
+  training_user_name_formatter_container:
+    class: Training\Bundle\UserNamingBundle\Container\UserNamePartsContainer
```

might be simplified to
```suggestion
  Training\Bundle\UserNamingBundle\Container\UserNamePartsContainer: ~
```